horizons/main.py

```diff
@@ -557,13 +557,10 @@ def edit_game_map(saved_game_name):
 	@return: bool, whether loading succeeded
 	"""
 	saved_games = SavegameManager.get_saves()
-	saved_game_path = _find_map(saved_game_name, saved_games)
-	if not saved_game_path:
+	map_name = _find_map(saved_game_name, saved_games)
+	if not map_name:
 		return False
 
-	accessor = SavegameAccessor(saved_game_path, False)
```

Comment:
Are you sure this is a fix? Isn't the idea of `--edit-game-map` to pass it a savegame, and use the map from that savegame in the editor? Does that still work?

I haven't tested it though, just curious why we wouldn't need to "parse" the SaveGameAccessor anymore.